src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -112,8 +114,12 @@ export class Conversations extends React.Component<Props, State> {
         initialListSize={10}
         scrollEventThrottle={10}
         onEndReachedThreshold={10}
+        enableEmptySections={true}
         refreshControl={
-          <RefreshControl refreshing={this.state.fetchingNextPage} onRefresh={() => this.fetchData(false)} />
+          <RefreshControl
+            refreshing={this.state.fetchingNextPage && this.state.initialLoadDone}
+            onRefresh={() => this.fetchData(false)}
```

Comment:
This is calling `fetchData` which in turn calls`forceFetch`  (https://github.com/artsy/emission/pull/649/files#diff-1a3b226b50eea831f2935ab045a6415aL94). The list updates fine so I think we're good. Home has nested listed in each row so that could also be why it needs more code to refresh everything.